### DISCUSSION TITLE
JavaScript Guide Index Page

### DISCUSSION THREAD

**Discussion by jhammen** [mdn/mdn-community#690] [Content]
Hi all, I am enjoying the JavaScript Guide but I wonder if the index page is a little out of date 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide

In particular the chapters and subheadings don't line up exactly with the current content, e.g. some top level chapter divisions are missing and some 2nd level headings are listed instead. Also for instance the "Regular expressions" chapter is listed as a subhead of "Text formatting"

Is this page auto generated somehow? If not would there be interest in a PR that cleans up the chapter/subhead organization a little bit to match the current organization of the content?

**Comment by pransh15** [comment]
Hi @jhammen 👋
Thank you for initiating this discussion. I reviewed the sections on "Regular expressions" and "Text formatting," and it seems they are not subheadings. I found the section selector a bit confusing and wanted to confirm if this is correct. 😅

![Screenshot 2024-06-18 at 13 47 07](https://github.com/mdn/mdn-community/assets/18158596/50ffd2e4-3045-485d-be53-b4379a085ec4)
_Regular Expressions are selected._

![Screenshot 2024-06-18 at 13 47 38](https://github.com/mdn/mdn-community/assets/18158596/94a5a48b-0379-48b3-be2f-2f8c6010b801)
_Text Formatting is selected._

Forgive me if I've got it wrong, but I'd love to see a couple more examples of outdated content. Also, we're interested in a PR! 🙌

**Comment by jhammen** [comment]
Hi @pransh15, thank you for your reply, I agree the menu headings on the left of the page match the content, no issues there, my issue above has to do with the content of the index/TOC page itself:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide

For example on this page I see the "text formatting" chapter is marked up as an H2 header but the "regular expressions" chapter listed as a LI list item underneath this header. In some of the other sections the list items are 2nd level rather than 1st level divisions of the chapter.

So my question was how is the content of this TOC page generated? If it's done by hand a PR makes sense, otherwise maybe not :)

**Comment by Josh-Cena** [comment]
@jhammen Thanks for the report. This page is practically unmaintained (because I have no interest in maintaining this page). It's purely handwritten. I wonder if it's possible to replace it with the landing page format used in other pages (i.e. a dl of one page, one paragraph description) instead of linking to each section which is unmaintainable.
